Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -46,9 +46,34 @@ class SettingsViewController: UIViewController, ToolbarViewProtocol, UITableView
         signOutButton.addTarget(self, action: "didClickLogout", forControlEvents: UIControlEvents.TouchUpInside)
```

Comment:
Heh, nope...just made that convention up. We can change it to anything else if there's a more common convention, but I do think it's useful to mark them so it's clear how they're used.